org.scala-ide.sdt.core.tests/pom.xml

```diff
@@ -14,6 +14,30 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <!-- log4j without its dependencies -->
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.mail</groupId>
+          <artifactId>mail</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>javax.jms</groupId>
+          <artifactId>jms</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jdmk</groupId>
+          <artifactId>jmxtools</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jmx</groupId>
+          <artifactId>jmxri</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
I don't know, they are just moved and have been existing before my change

2016-01-05 14:13 GMT+01:00 Simon Schäfer <notifications@github.com>:

> In org.scala-ide.sdt.core.tests/pom.xml
> <https://github.com/scala-ide/scala-ide/pull/1054#discussion_r48842916>:
>
> > +          <groupId>javax.mail</groupId>
> > +          <artifactId>mail</artifactId>
> > +        </exclusion>
> > +        <exclusion>
> > +          <groupId>javax.jms</groupId>
> > +          <artifactId>jms</artifactId>
> > +        </exclusion>
> > +        <exclusion>
> > +          <groupId>com.sun.jdmk</groupId>
> > +          <artifactId>jmxtools</artifactId>
> > +        </exclusion>
> > +        <exclusion>
> > +          <groupId>com.sun.jmx</groupId>
> > +          <artifactId>jmxri</artifactId>
> > +        </exclusion>
> > +      </exclusions>
>
> Why do we need this exclusions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/1054/files#r48842916>.
>